libtransmission/rpcimpl.cc

```diff
@@ -869,6 +871,17 @@ char const* set_file_priorities(tr_torrent* tor, tr_priority_t priority, tr_vari
     return nullptr; // no error
 }
 
+char const* set_sequential_download_from_piece(tr_torrent* tor, tr_piece_index_t piece)
```

Comment:
*(Optional)* Prefer to use references over pointers for new functions.

```suggestion
char const* set_sequential_download_from_piece(tr_torrent& tor, tr_piece_index_t piece)
```